externalAPI/rest/rest.go

```diff
@@ -118,7 +118,12 @@ func (c *APIClient) buildRequest(method string, path string, body io.Reader, par
 	return req
 }
 
-func BuildBody(data map[string]interface{}) io.Reader {
+func BuildJsonBody(data map[string]interface{}) io.Reader {
+	body, _ := json.Marshal(&data)
+	return bytes.NewReader(body)
+}
+
+func BuildStringBody(data interface{}) io.Reader {
```

Comment:
What's the difference with BuildJsonBody?

Reply:
fixed